client/js/lounge.js

```diff
@@ -465,6 +465,23 @@ $(function() {
 		return false;
 	});
 
+	sidebar.on("click", ".add", (e) => {
+		e.preventDefault();
+		const id = $(e.target).data("id");
```

Comment:
Same here with ids, you can just go to parent and find the relevant form. I'm suggesting this because we don't use this `id` approach anywhere else.